src/app/components/CurrencyAsset/index.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ *
+ * CurrencyAssets
+ *
+ */
+import React from 'react';
+import styled from 'styled-components/macro';
+
+import { Asset } from 'types/asset';
+
+import { AssetsDictionary } from '../../../utils/dictionaries/assets-dictionary';
+
+interface CurrencyProps {
+  assetImg?: any;
+  assetSymbol: Asset;
+}
+const symbolMap = {
+  [Asset.RBTC]: 'rBTC',
+  [Asset.USDT]: 'rUSDT',
+};
+// function renderCustomSymbol(currency: string) {
+//   return (
+//     <div className="d-flex flex-row">
+//       <Small>{currency.charAt(0)}</Small>
+//       {currency.slice(1)}
+//     </div>
+//   );
+// }
+function getAssetSymbol(asset: Asset) {
+  if (symbolMap.hasOwnProperty(asset)) {
+    return symbolMap[asset];
+  }
+  return AssetsDictionary.get(asset).symbol;
+}
+export function CurrencyAssets(props: CurrencyProps) {
```

Comment:
Current name suggest it's a list of currencies, change it

```suggestion
export function AssetRenderer(props: CurrencyProps) {
```